test/reporters/xml.test.coffee

```diff
@@ -52,6 +52,10 @@ describe "xml reporter", ->
         path: "test/fixtures/"
         languages: [language]
         reporter: 'xml'
+        # override what is in base .cpd.yaml
```

Comment:
Because the config file now supplies defaults, passed options must override. Previously, the config file was disregarded if options were passed.